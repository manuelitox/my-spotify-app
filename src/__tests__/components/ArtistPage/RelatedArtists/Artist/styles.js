import React from 'react'
import { shallow } from 'enzyme'

import StyledArtist from 'components/ArtistPage/RelatedArtists/Artist/styles'

describe('Component | StyledArtist', () => {
  let component 
  beforeEach( () => {
    component = shallow(
      <StyledArtist />
    )
  })
  it('should render a component', () => {
    expect(component).toMatchSnapshot()
  })
})