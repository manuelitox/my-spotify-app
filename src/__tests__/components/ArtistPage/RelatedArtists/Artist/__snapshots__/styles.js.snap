// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component | StyledArtist should render a component 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
  display: grid;
  grid-template-columns: 40px 1fr;
  grid-gap: 20px;
  position: relative;
  h4 {
    align-self: center;
    font-weight: 500;
    margin: 0;
  }
  img {
    border-radius: 4px;
  }
  a {
    bottom: 0;
    left: 0;
    height: 100%;
    position: absolute;
    width: 100%;
    z-index: 1;
  }
",
        ],
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
